src/backend/lib/db_helpers/case_manager.ts

```diff
@@ -62,15 +62,24 @@ export async function sendInviteEmail(
   fromEmail: string,
   toEmail: string,
   first_name: string,
-  caseManagerName: string,
+  case_manager_name: string,
```

Comment:
Should we be consistent with snake case vs. Pascal case? I'm thinking that maybe we can reserve snake case for what's going directly into the database.

I think there should be an abstraction layer, always at the same level in the codebase where you may have stuff like:

```
first_name: firstName,
email: email.toLowerCase(),
case_manager_name: caseManagerName
```

Reply:
hmm yeah and it's interesting that `createPara` and `sendInviteEmail` both have `*Email` but in diff cases, although that was already there prior to this PR.

not as clear to me about which case ought to belong where, but perhaps worth creating a new issue (or discussion!) to address codebase-wide? @canjalal

Reply:
thanks both, I think we need to open up discussion in eng for this.